debian/librados-dev.install

```diff
@@ -10,5 +10,4 @@ usr/include/rados/page.h
 usr/include/rados/rados_types.h
 usr/include/rados/rados_types.hpp
 usr/lib/librados.so
-usr/lib/librados_tp.so
```

Comment:
i think librados_tp.so is the tracing point library for librados, which is a client only library. and ceph-osd does not necessarily install librados. so, with this change, librados will be installed without librados_tp.so. is this expected?

Reply:
when do ceph-deploy install, it will install librados.so and librados_tp.so ( just as the setting in librados-dev.install) to /usr/lib.  Anyway they will be installed, and in some case, we run client and osd in same server for developing and testing, that will help.

Reply:
> we run client and osd in same server for developing and testing, that will help

@liu-chunmei what i mean is, this does not work for users who do not install OSD and client on the same server.